src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -333,6 +333,7 @@ public Claim paidInFull(String externalId, PaidInFull paidInFull, String authori
         this.caseRepository.paidInFull(claim, paidInFull, authorisation);
         Claim updatedClaim = getClaimByExternalId(externalId, authorisation);
         this.eventProducer.createPaidInFullEvent(updatedClaim);
+        appInsights.trackEvent(AppInsightsEvent.PAID_IN_FULL, REFERENCE_NUMBER, claim.getReferenceNumber());
```

Comment:
Hey, can you confirm how I would verify this? Cheers

verify(appInsights, once()).trackEvent(???);

Reply:
It would be like:

```
verify(appInsights, once()).trackEvent(eq(PAID_IN_FULL), eq(REFERENCE_NUMBER), eq(claim.getReferenceNumber()));
```

Reply:
Or `verify(appInsights).trackEvent(PAID_IN_FULL, REFERENCE_NUMBER, claim.getReferenceNumber())` since `once()` is redundant and you don't need the `eq()`s if it's used for every parameter.